assets/app/util/documentStrategy.js

```diff
@@ -0,0 +1,50 @@
+import { decodeB64 } from './encoding';
+
+const frontmatterDelimiterRegex = /^---\n([\s\S]*?)---\n/;
+
+const initializeYml = content => {
+  return {
```

Comment:
There are a lot of return statements but I find it pretty easy to read since its all singe functions that are mostly pure